spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  describe 'creates a post for a user' do
+    it 'displays the post created' do
+      # display a page
+      visit user_posts_path(1)
+      # add post
+      sleep(1)
+      assert_text 'Create a new post'
+      click_button('Create a new post')
+
+      #   # fill the form with text
+      sleep(1)
+      fill_in 'post[Title]', with: 'Yet another post..'
+      fill_in 'post[Text]', with: 'This is another post hope you like reading..'
+
+      sleep(1)
+      click_button 'Create Post'
+      sleep(2)
+    end
+  end
+end
```

Comment:
- [ ]  as part of the requirements, kindly add these missing tests for the post index :


* Tests that asserts you can see the user's profile picture.
* Tests that asserts you can see the user's username.
* Tests that asserts you can see the number of posts the user has written.
* Tests that asserts you can see a post's title.
* Tests that asserts you can see some of the post's body.
* Tests that asserts you can see the first comments on a post.
* Tests that asserts you can see how many comments a post has.
* Tests that asserts you can see how many likes a post has.
* Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
* Tests that redirects to the post's show page when you click on a post.